app/views/works/new.html.erb

```diff
@@ -0,0 +1,20 @@
+<h1 class="h2 mb-3">Untitled deposit</h1>
+ <div class="d-flex tabbable-panes">
```

Comment:
I didn't componentize this because as of now only used in one place and doesn't involve any logic.